www/downloads/index.html

```diff
@@ -4,6 +4,83 @@
 ---
 <div class="block">
 <h2 id="downloads">Downloads</h2>
+<h3>DDNet 16.0</h3>
+<div class="dlfiles">
+  2022-03-28<br/><br/>
+  <ul>
+    <li><a href="DDNet-16.0-win32.zip">Windows&nbsp;32bit</a></li>
+    <li><a href="DDNet-16.0-win64.zip">Windows&nbsp;64bit</a></li>
+    <li><a href="DDNet-16.0-linux_x86.tar.xz">Linux&nbsp;x86</a></li>
+    <li><a href="DDNet-16.0-linux_x86_64.tar.xz">Linux&nbsp;x86-64</a></li>
+    <li><a href="DDNet-16.0-macos.dmg">macOS (arm64&nbsp;&amp;&nbsp;x86-64)</a></li>
+    <li><a href="DDNet-16.0.tar.xz">Source</a></li>
+  </ul>
+</div>
+<div class="dlinfo">
+  <strong>Changelog</strong>
+  <ul>
+    <li>[Client] <strong>New Vulkan graphics renderer for more FPS (enable in Graphics Settings)</strong> [<strong>Jupeyy</strong>]</li>
+    <li>[Client] <strong>Improved OpenGL graphics renderers</strong> [<strong>Jupeyy</strong>]</li>
```

Comment:
Improved FFmpeg video renderer is the important thing actually :D

Like Demo to Video rendering